### PR TITLE
Add support for material design icons

### PR DESCRIPTION
**Pull request by ma12-co**
Add slot for Material design icons to the actions 

https://user-images.githubusercontent.com/26852655/108392906-0f30df80-720b-11eb-873a-a0421f994390.mp4

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/Actions/Actions.vue [hunk 1]
```diff
@@ -82,6 +82,20 @@ https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-action
 ```
 
 ### Custom icon slot
+To be used with `vue-material-design-icons` only. For icon classes use the `default-icon` slot.
+It can be used with one or multiple actions.
+```
+<Actions>
+	<MicrophoneOff
```
Comment (nickvergessen) [inline src/components/Actions/Actions.vue:89]: This looks weird. Should be inside an ActionButton or something?

Comment (ma12-co) [reply src/components/Actions/Actions.vue:89]: Why does it look weird to you?  And no the actionbutton is something else

Comment (nickvergessen) [reply src/components/Actions/Actions.vue:89]: I missed the `slot` line

**Review by skjnldsv — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

**Pull request merged**